extra/aws/external-worker-aws-cf-template.yaml

```diff
@@ -245,7 +249,13 @@ Resources:
   WorkersRole:
     Type: "AWS::IAM::Role"
     Properties:
-      #RoleName: "cycloid-ci-workers"
+      RoleName:
+        Fn::Join:
+          - '-'
+          - - !Ref AWS::StackName
+            - !Ref RoleTag
```

Comment:
I feel most of the time the stack name will be like cycloid-workers

Not sure we need roleTag inside the name to avoid cycloid-worker-worker

Reply:
to match the name on concourse side